src/lib/dialog/dialog.ts

```diff
@@ -67,6 +69,7 @@ export class MatDialog {
   private _openDialogsAtThisLevel: MatDialogRef<any>[] = [];
   private _afterAllClosedAtThisLevel = new Subject<void>();
   private _afterOpenAtThisLevel = new Subject<MatDialogRef<any>>();
+  private _ariaHiddenElements = new Map<Element, string|null>();
```

Comment:
`WeakMap` as a precaution against memory leaks?

Reply:
We can't use a `WeakMap`, because we need to be able to iterate over the elements.